src/components/Home.jsx

```diff
@@ -1,16 +1,32 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
 
 import SearchField from './SearchField';
 import SearchableDropdown from './SearchableDropdown';
 import CurrentWeather from './CurrentWeather';
 
 import useLocalStorage from './CustomHooks/useLocalStorage';
 import { ENTER_KEYCODE, MAX_ITEMS_LENGTH } from '../common/constants';
+import { groupQueryString } from '../common/helpers';
+import { refreshAccessToken } from '../common/auth';
 
 const Home = () => {
-  const [searchWord, setSearchWord] = useState('');
+  const history = useHistory();
+  const { search } = useLocation();
+
+  const groupedQueryStrings = groupQueryString(search);
+  const cityName = groupedQueryStrings.city?.replace(/%20/g, ' ');
```

Comment:
no magic strings please:) Create a function, say, trim(), or removeSpaces(), or whatever else. Or at least create a constant for your regExp. 

And again... do you really need to use query params to pass around your data ?